spec/features/user_registers_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+feature 'user register a new account', { js: true, vcr: true } do
```

Comment:
I did not say it has to be wrapped in an array. `feature 'user register a new account', :js, :vcr do` like that.